src/game.cpp

```diff
@@ -363,72 +442,58 @@ PointedThing getPointedThing(Client *client, Hud *hud, const v3f &player_positio
 
 	// That didn't work, try to find a pointed at node
 
-	v3s16 pos_i = floatToInt(player_position, BS);
-
-	/*infostream<<"pos_i=("<<pos_i.X<<","<<pos_i.Y<<","<<pos_i.Z<<")"
-			<<std::endl;*/
-
-	s16 a = d;
-	s16 ystart = pos_i.Y - (camera_direction.Y < 0 ? a : 1);
-	s16 zstart = pos_i.Z - (camera_direction.Z < 0 ? a : 1);
-	s16 xstart = pos_i.X - (camera_direction.X < 0 ? a : 1);
-	s16 yend = pos_i.Y + 1 + (camera_direction.Y > 0 ? a : 1);
-	s16 zend = pos_i.Z + (camera_direction.Z > 0 ? a : 1);
-	s16 xend = pos_i.X + (camera_direction.X > 0 ? a : 1);
-
-	// Prevent signed number overflow
-	if (yend == 32767)
-		yend = 32766;
-
-	if (zend == 32767)
-		zend = 32766;
+	v3s16 pointed_pos(0, 0, 0);
 
-	if (xend == 32767)
-		xend = 32766;
+	v3f start = camera_position / BS;
+	v3f end = start + (camera_direction * d);
+
+	Interpolator interpolator(start, end);
+
+	std::vector<v3s16> nodes;
+	std::vector<aabb3f> boxes;
+	while (!interpolator.IsDone())

```

Comment:
The code style rule is: braces always on the same line, except for function braces. I've seen this at multiple places in the PR, but only make one comment about it.